pkg/api/routes.go

```diff
@@ -452,7 +452,17 @@ func (rh *RouteHandler) UpdateManifest(response http.ResponseWriter, request *ht
 		return
 	}
 
-	digest, err := imgStore.PutImageManifest(name, reference, mediaType, body)
+	var mandatoryAnnotations []string
+
+	var lintEnabled bool
+
+	if rh.c.Config.Extensions != nil && rh.c.Config.Extensions.Lint != nil &&
```

Comment:
Can we move this into the "storage" layer instead so that none of the pkg/storage/storage.go API signatures need to change?